tests/redirects/map_301.py

```diff
@@ -310,3 +310,29 @@
     url_test("/samples/canvas-tutorial/images/wallpaper.png",
              "https://mdn.mozillademos.org/files/222/Canvas_createpattern.png"),
 ))
+
+LEGACY_URLS = flatten((
+    # bug 1362438
+    url_test('/index.php', status_code=404),
+    url_test('/index.php?title=Special:Recentchanges&feed=atom',
+             status_code=404),
+    url_test('/index.php?title=En/HTML/Canvas&revision=11',
+             status_code=404),
+    url_test('/index.php?title=En/HTML/Canvas&revision=11',
+             status_code=404),
+    url_test('/patches', status_code=404),
+    url_test('/patches/foo', status_code=404),
+    url_test('/web-tech', status_code=404),
+    url_test('/web-tech/feed/atom/', status_code=404),
+    url_test('/web-tech', status_code=404),
+    url_test('/web-tech/feed/atom/', status_code=404),
```

Comment:
It looks like lines 327 and 328 are copies of 325 and 326.

Reply:
😞 yes those are dupes